.github/workflows/test.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       run: |
         go get -t -v ./...
         go test -v -race ./...
-        cd hack/tools && go test -v -race ./...
+        cd hack/tools && go test -v -race ./... && cd ../..
```

Comment:
this will fail to `cd ../..` if any test fails, ensuring the next step fails, too

Reply:
I was using `(cd hack/tools && go test -v -race ./...)` earlier, but that failed in Windows. The shell there doesn't seem to support sub-shells.

Is there an absolute path that can be used here?

Reply:
`cd hack/tools && go test -v -race ./... ; cd ../..` ?

`sh -c "cd hack/tools && go test -v -race ./..."` ?

FWIW https://github.com/golang/go/issues/50332